src/cellfinder_core/detect/filters/volume/structure_detection.py

```diff
@@ -45,34 +45,20 @@ def traverse_dict(d: Dict[T, T], a: T) -> T:
         return a
 
 
-def get_structure_centre(structure: Sequence[Point]) -> Point:
+@njit
+def get_structure_centre(structure: np.ndarray) -> np.ndarray:
```

Comment:
It looks like this was written with cython in mind, and can now be re-written much more simply as just 
```python
return np.round(np.mean(structure, axis=0))
```
?

Reply:
[Numba doesn't support `axis`](https://numba.pydata.org/numba-doc/dev/reference/numpysupported.html#calculation) but I've simplified along these lines.